.github/workflows/demo.yml

```
@@ -0,0 +1,146 @@
+  LANG: "en_US.UTF-8"
+  ATMOS_LOGS_LEVEL: "Info"
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get Atmos version
+        id: get-version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)